requirements/audio.txt

```diff
@@ -6,3 +6,6 @@ pesq >=0.0.4, <0.0.5
 pystoi >=0.3.0, <0.5.0
 torchaudio >=0.10.0, <2.5.0
 gammatone >=1.0.0, <1.1.0
+librosa >=0.9.0, <0.11.0
+onnxruntime >=1.12.0, <1.19
```

Comment:
Shouldn't we consider the possibility to use `onnxruntime-gpu` to utilize GPUs for the computation?

Reply:
I may say that would be an optimized version, so a user would uninstall the normal and install the GPU version so that we can mention it in readme somewhere... also do it in the CI docker images :flamingo: